code/modules/projectiles/gun.dm

```diff
@@ -173,7 +173,6 @@
 					PBT.throw_at(throw_target, pb_knockback, 2)
 			else if(!tk_firing(user))
 				user.visible_message(span_danger("[user] fires [src]!"), \
-								span_danger("You fire [src]!"), \
```

Comment:
I don't think this removes the message, I think this'll just make it so "You hear a gunshot!" is the new message they see

Reply:
It did but I manage to figure out the proper way to make these be quiet to the user.
Thanks.